doc/nrf/releases_and_maturity/releases/release-notes-3.0.0-preview1.rst

```diff
@@ -625,7 +625,9 @@ Common Application Framework
 Debug libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`cpu_load` library:
+
+  * Added prefix ``NRF_`` to all Kconfig options (e.g. :kconfig:option:`CONFIG_NRF_CPU_LOAD`) for that module to avoid conflict with Zephyr Kconfig option with the same name.
```

Comment:
```suggestion
  * Added prefix ``NRF_`` to all Kconfig options (for example, :kconfig:option:`CONFIG_NRF_CPU_LOAD`) to avoid conflict with Zephyr Kconfig option with the same name.
```